docs/tools/mockery/generate-mocks-with-mockery.md

```diff
@@ -20,3 +20,42 @@ type AccessCodeClaimer interface {
     ClaimAccessCode(code string, serviceMemberID uuid.UUID) (*models.AccessCode, *validate.Errors, error)
 }
 ```
+
+## Using Mocks
```

Comment:
This section is pretty much the same as before, I just moved it from the service object docs to this page since it fits better here.